lib/rucio/core/did.py

```diff
@@ -2052,52 +2073,98 @@ def create_did_sample(input_scope, input_name, output_scope, output_name, accoun
 
 
 @transactional_session
-def __resolve_bytes_length_events_did(scope: str, name: str, session: Session) -> Tuple[int, int, int]:
+def __resolve_bytes_length_events_did(
+        did: models.DataIdentifier,
+        dynamic_depth: "DIDType" = DIDType.FILE,
```

Comment:
`dynamic_depth` default value is sometimes `None`, here `DIDType.File`. We should maybe stick to one.

Reply:
I don't think this is problematic. It's a private function and its name clearly states that we will resolve the size of whatever it's passed in.